recipes/xrandr.mdwn

```diff
@@ -0,0 +1,16 @@
+# Multiple Monitor Helper
+
+This page is intended to show how to user `xrandr.lua` script to help the way you deal with multiple monitors.
+Using this snippet you can set a keybinding where you swap to all `(Number of Monitors)!` screen setups.
```

Comment:
So I guess the ! is meant as faculty, but I would also guess that for the average user it would be more helpful to give the combinatoric explanation instead of just the number of possibilities. So "all possible horizontal arrangements of monitors"?